Makefile

```diff
@@ -1,9 +1,18 @@
-.PHONY: image test
+.PHONY: image test release
 
 IMAGE_NAME ?= codeclimate/codeclimate-sonar-php
+RELEASE_REGISTRY ?= codeclimate
+
+ifndef RELEASE_TAG
+override RELEASE_TAG = latest
+endif
```

Comment:
```suggestion
RELEASE_TAG ?= latest
```

Reply:
isn't that simpler ?

Reply:
this does not work when providing an empty string. Makefile takes empty strings as not null, so it'll end up not tagging the image as latest. The ifndef workaround lets us provide an empty string that will still be replaced with latest

Reply:
@dantevvp 

> this does not work when providing an empty string.

That's expected isn't it ? Why would you run `make release RELEASE_TAG=""`  ?